examples/integration-tutorials/sending-a-simple-message-to-a-service/sending-a-simple-message-to-a-service/simple_service.bal

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+
+listener http:Listener httpListener = new(9090);
+
+// Health Care Management is done using an in-memory map.
+map<json> appoinmentMap = {};
+
+// RESTful service.
+@http:ServiceConfig { basePath: "/hospitalmgt" }
+service hospitalMgt on httpListener {    
+    
+    // Resource that handles the HTTP GET requests  
+    @http:ResourceConfig {
+        methods: ["GET"],
+        path: "/medicalreservation/{patientID}"
+    }
+    resource function getAppoinmentDetails(http:Caller caller, http:Request req, string patientID) {
+        // Find the requested appoinment from the map and retrieve it in JSON format.
+        json? payload = appoinmentMap[patientID];
+        http:Response response = new;
+        if (payload == null) {
+            payload = "Medical reservation : " + patientID + " cannot be found.";
+        }
+        // Set the JSON payload in the outgoing response message.
+        response.setJsonPayload(untaint payload);
+        // Send response to the client.
+        var result = caller->respond(response);
+        if (result is error) {
+            log:printError("Error sending response", err = result);
+        }
+    }
+
+    // Resource that handles the HTTP POST requests
+    @http:ResourceConfig {
+        methods: ["POST"],
+        path: "/medicalreservation"
+    }
+    resource function addAppoinmentDetails(http:Caller caller, http:Request req) {
+        http:Response response = new;
+        var patientReq = req.getJsonPayload();
+        if (patientReq is json) {
+            string patientID = patientReq.Appoinment.ID.toString();
+            appoinmentMap[patientID] = patientReq;
+
+            var payload = req.getJsonPayload();
+            // Create response message.
+            json jsonpayload = {};
+            if (payload is json) {
+            jsonpayload = untaint payload;
+                } else {
+            jsonpayload = "error";
+            }
+            response.setJsonPayload(untaint jsonpayload);
```

Comment:
since we already have `patientReq` which is a confirmed json, we should be able to just send the payload as it is. 
`response.setJsonPayload(untaint patientReq);` 

Lines 62 - 69 can be removed I think.

Reply:
Fixed.